nemo/collections/nlp/modules/common/huggingface/albert.py

```diff
@@ -27,6 +27,10 @@ class AlbertEncoder(AlbertModel, BertModule):
     Wraps around the Huggingface transformers implementation repository for easy use within NeMo.
     """
 
+    def __init__(self) -> None:
+        AlbertModel.__init__()
```

Comment:
this looks strange to me. @ekmb what do you think?

Reply:
Oleksii, I am trying to fix the problem that RTD/Sphinx is having with:
```
WARNING: autodoc: failed to import class 'nlp.models.GLUEModel' from module 'nemo.collections'; the following exception was raised:
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/nvidia-nemo/envs/main/lib/python3.7/site-packages/sphinx/ext/autodoc/importer.py", line 154, in import_module
    __import__(modname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/nvidia-nemo/checkouts/main/nemo/collections/nlp/__init__.py", line 15, in <module>
    from nemo.collections.nlp import data, models, modules
  File "/home/docs/checkouts/readthedocs.org/user_builds/nvidia-nemo/checkouts/main/nemo/collections/nlp/models/__init__.py", line 15, in <module>
    from nemo.collections.nlp.models.glue_benchmark.glue_benchmark_model import GLUEModel
  File "/home/docs/checkouts/readthedocs.org/user_builds/nvidia-nemo/checkouts/main/nemo/collections/nlp/models/glue_benchmark/__init__.py", line 15, in <module>
    from nemo.collections.nlp.models.glue_benchmark.glue_benchmark_model import GLUEModel
  File "/home/docs/checkouts/readthedocs.org/user_builds/nvidia-nemo/checkouts/main/nemo/collections/nlp/models/glue_benchmark/glue_benchmark_model.py", line 28, in <module>
    from nemo.collections.nlp.models.nlp_model import NLPModel
  File "/home/docs/checkouts/readthedocs.org/user_builds/nvidia-nemo/checkouts/main/nemo/collections/nlp/models/nlp_model.py", line 25, in <module>
    from nemo.collections.nlp.modules import MegatronBertEncoder
  File "/home/docs/checkouts/readthedocs.org/user_builds/nvidia-nemo/checkouts/main/nemo/collections/nlp/modules/__init__.py", line 16, in <module>
    from nemo.collections.nlp.modules.common import (
  File "/home/docs/checkouts/readthedocs.org/user_builds/nvidia-nemo/checkouts/main/nemo/collections/nlp/modules/common/__init__.py", line 18, in <module>
    from nemo.collections.nlp.modules.common.huggingface import (
  File "/home/docs/checkouts/readthedocs.org/user_builds/nvidia-nemo/checkouts/main/nemo/collections/nlp/modules/common/huggingface/__init__.py", line 15, in <module>
    from nemo.collections.nlp.modules.common.huggingface.albert import AlbertEncoder
  File "/home/docs/checkouts/readthedocs.org/user_builds/nvidia-nemo/checkouts/main/nemo/collections/nlp/modules/common/huggingface/albert.py", line 25, in <module>
    class AlbertEncoder(AlbertModel, BertModule):
  File "/home/docs/checkouts/readthedocs.org/user_builds/nvidia-nemo/envs/main/lib/python3.7/abc.py", line 126, in __new__
    cls = super().__new__(mcls, name, bases, namespace, **kwargs)
TypeError: Cannot create a consistent method resolution
order (MRO) for bases _MockObject, BertModule
```

As you can see, it is failing on MRO, which suggests something with resolution the order of calling the inits. 

So trying to call them manually. 

Most importantly: this is still not working (hence `draft` status of PR), will poke you when done, ok?